cmd/pipeline/destroy.go

```diff
@@ -101,8 +101,10 @@ func ExecuteDestroyPipeline(ctx context.Context, opts *DestroyOptions) error {
 		return nil
 	}
 
-	if err := waitUntilDestroyed(ctx, opts.Name, resp.Action, opts.Timeout); err != nil {
-		return err
+	if resp != nil {
```

Comment:
I think it is odd to have a nil response without any error if the pipeline is not found. I think a better approach is to return a specific error within `destroyPipeline` (it seems it is only being used here) and let the client decide what to do with it. 

In this case, we would check the error type to wait or not until destroyed. Anyways, a minor comment

Reply:
Will do and add some test mocking the okteto API